shell/platform/darwin/macos/framework/Source/FLEViewController.mm

```diff
@@ -612,11 +615,34 @@ - (void)handleMethodCall:(FlutterMethodCall*)call result:(FlutterResult)result {
   if ([call.method isEqualToString:@"SystemNavigator.pop"]) {
     [NSApp terminate:self];
     result(nil);
+  } else if ([call.method isEqualToString:@"Clipboard.getData"]) {
+    result([self getClipboardData:call.arguments]);
+  } else if ([call.method isEqualToString:@"Clipboard.setData"]) {
+    [self setClipboardData:call.arguments];
+    result(nil);
   } else {
     result(FlutterMethodNotImplemented);
   }
 }
 
+#pragma mark - ClipboardData
+
+- (NSDictionary*)getClipboardData:(NSString*)format {
+  NSPasteboard* pasteboard = [NSPasteboard generalPasteboard];
+  if (!format || [format isEqualToString:@(kTextPlainFormat)]) {
```

Comment:
Why is no format equivalent to text? If that's intentional, it needs a comment explaining why.